lib/tasks/update_serachable_content_for_vacancies_with_visa_sponsorship.rake

```diff
@@ -0,0 +1,8 @@
+namespace :vacancy do
+  desc "Update searchable content for vacancies with visa sponsorship"
+  task update_searchable_content: :environment do
+    Vacancy.where(visa_sponsorship_available: true).find_each do |vacancy|
+      vacancy.update!(searchable_content: vacancy.generate_searchable_content)
+    end
+  end
+end
```

Comment:
I added this rake task to update searchable_content for each vacancy which had visa_sponsorship_available set to true because otherwise they would not have "visa sponsorship" in their searchable_content until they were edited and would not show up in the filtered results when a user entered "visa" in the keyword search box.